build/ci.go

```diff
@@ -272,6 +274,56 @@ func doTest(cmdline []string) {
 	build.MustRun(gotest)
 }
 
+func doCover(cmdline []string) {
+	var (
+		parallel   = flag.Int("p", 0, "The number of parallel coverage test executions (default: the number of CPUs available)")
+		excludes   = flag.String("exclude", "", "Comma-separated top-level directories to be excluded in coverage test")
+		outputFile = flag.String("coverprofile", "coverage.out", "The coverage profile file will be generated by coverage test")
+	)
+	flag.CommandLine.Parse(cmdline)
+	env := build.Env()
+
+	packages := []string{"./..."}
+	if len(flag.CommandLine.Args()) > 0 {
+		packages = flag.CommandLine.Args()
+	}
+	packages = build.ExpandPackagesNoVendor(packages)
+	if *excludes != "" {
+		packages = build.ExcludePackages(packages, strings.Split(*excludes, ","))
+	}
+
+	coverPackages := []string{"./..."}
+	coverExcludes := []string{
+		"github.com/klaytn/klaytn/tests",
```

Comment:
How about adding "github.com/klaytn/klaytn/tests/benchmarks"?

Reply:
`vender` is already excluded?

Reply:
@jimni1222 I replaced the exclude pattern to `/test`.
@KimKyungup `vendor` is excluded in `ExpandPackagesNoVendor`.